README.md

```diff
@@ -39,31 +39,36 @@ Do this with a path file in your venv. OpenAI evals needs to import it.
 Create a venv with
 
 `python3.9 -m venv venv`
+- Note for Mac users: if you dont have python3.9 install it with ` brew install python@3.9`
```

Comment:
This is a good point.
Might be worth adding in
apt install python3.9 for ubuntu too. 
You also likely have to install python3.9-venv too